sortedm2m/fields.py

```diff
@@ -316,31 +311,9 @@ def get_rel_to_model_and_object_name(self, klass):
             to_object_name = to_model._meta.object_name
         return to_model, to_object_name
 
-    def get_intermediate_model_sort_value_field_default(self, klass):
-        def default_sort_value(name):
```

Comment:
Why do you remove the callable here to create the default? What's the benefit of calculating in the `_add_items` method?